pages/api/auth/verify-account.ts

```diff
@@ -0,0 +1,58 @@
+import { generateToken, validateEmail } from '@/lib/common';
```

Comment:
Can we rename this API to `/resend-email-token`?

Reply:
done